src/lib/utils/options/readers/tsconfig.ts

```diff
@@ -26,6 +26,30 @@ export class TSConfigReader extends OptionsComponent
     private static OPTIONS_KEY:string = 'tsconfig';
 
 
+    private readConfigFile(event:DiscoverEvent, fileName:string) {
+        if (!FS.existsSync(fileName)) {
```

Comment:
I feel like it would be legacy and we should be able to tidy it up instead of trying to build our own extends. However, if there's too much to correct, let me know.